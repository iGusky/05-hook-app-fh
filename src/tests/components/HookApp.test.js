import { shallow } from 'enzyme'
import HookApp from '../../HookApp'

describe('pruebas de <HookApp />', () => {

    const wrapper = shallow(<HookApp />)

    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})