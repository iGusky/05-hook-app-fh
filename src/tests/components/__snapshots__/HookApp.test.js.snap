// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas de <HookApp /> debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
